src/bitcoin-cli.cpp

```diff
@@ -148,7 +148,11 @@ static void http_request_done(struct evhttp_request *req, void *ctx)
 UniValue CallRPC(const string& strMethod, const UniValue& params)
 {
     std::string host = GetArg("-rpcconnect", DEFAULT_RPCCONNECT);
-    int port = GetArg("-rpcport", BaseParams().RPCPort());
+
+    int portTmp = GetArg("-rpcport", BaseParams().RPCPort());
```

Comment:
Nit: GetArg returns `int64_t`. I think the goal of the pull was not to cast without check?